object.c

```diff
@@ -4143,9 +4143,9 @@ f_sprintf(int c, const VALUE *v, VALUE _)
  *
  *  === What's Here
  *
- *  These are the methods defined for \BasicObject:
+ *  These are the methods defined for BasicObject:
  *
- *  - ::new: Returns a new \BasicObject instance.
+ *  - ::new: Returns a new BasicObject instance.
```

Comment:
This is documentation for the BasicObject class.

Reply:
Fixed.